cmd/commands/mounts.go

```diff
@@ -0,0 +1,48 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/Shopify/kubeaudit/auditors/mounts"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+const sensitivePathsFlagName = "paths"
+
+var mountsConfig mounts.Config
+
+var mountsCmd = &cobra.Command{
+	Use:   "mounts",
+	Short: "Audit containers that mount sensitive paths",
+	Long: fmt.Sprintf(`This command determines which containers mount sensitive host paths. If no paths list is provided, the following 
+paths are used:
+%s
+
+A WARN result is generated when a container mounts one or more paths specified with the '--paths' argument.
+
+Example usage:
+kubeaudit mounts --paths "%s"`, formatPathsList(), strings.Join(mounts.DefaultSensitivePaths[:3], ",")),
+	Run: func(cmd *cobra.Command, args []string) {
+		runAudit(mounts.New(mountsConfig))(cmd, args)
+	},
+}
+
+func init() {
+	RootCmd.AddCommand(mountsCmd)
+	setPathsFlags(mountsCmd)
+}
+
+func setPathsFlags(cmd *cobra.Command) {
+	cmd.Flags().StringSliceVarP(&mountsConfig.SensitivePaths, sensitivePathsFlagName, "s", mounts.DefaultSensitivePaths,
```

Comment:
Another tiny thing, and more of a question. How about `p` rather than `s` for the shorthand?
```suggestion
	cmd.Flags().StringSliceVarP(&mountsConfig.SensitivePaths, sensitivePathsFlagName, "p", mounts.DefaultSensitivePaths,
```

Reply:
Yes, that was my initial idea but `-p` is already used as a shorthand for the `--format` argument (I guess it comes from `pretty`?)
```
  -p, --format string        The output format to use (one of "pretty", "logrus", "json") (default "pretty")
```

So I fall back on `s`  for "**S**ensitive paths" but it's indeed not really obvious

Reply:
oh, yeah! you're right! perhaps `sp`? not sure...it's optional. We can always revisit that later if someone gives use feedback saying it's confusing.